src/content/en/2020/seo.md

```diff
@@ -23,3 +23,835 @@ queries: 07_SEO
+- Although self-referencing canonical tags are sometimes used, these aren’t a requirement.
+
+[Last year](../2019/seo#canonicalization), it was identified that 48.34% of mobile home pages were using a canonical tag. This year the number of mobile home pages featuring a canonical tag has grown to 53.61%.
+
+{{ figure_markup(
+  image="seo-presence-of-canonical-tag.png",
+  caption="Presence of Canonical Tag",
+  description="Bar chart showing presence of canonical tags. The majority of our analyzed web pages include a canonical tag, the main part being self-referential (47.88% for desktop and 45.31% for mobile). The percentage of canonicalized web pages is higher on mobile (8.45%) than on desktop (4.1%).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=1777344456format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sql"
+ )
+}}
+
+When analyzing this year’s mobile home pages canonical tag configuration, it was detected that 45.31% of them were self-referential and 8.45% were pointing to different URLs as the canonical ones.
+
+On the other hand, 51.85% of the desktop home pages were found to be featuring a canonical tag this year, with 47.88% being self-referential and 4.10% pointing to a different URL.
+
+Not only mobile home pages are featuring more canonical tags than desktop ones (53.61% vs. 51.85%) but there are more mobile homes pages canonicalizing to other URLs than their desktop counterparts (8.45% vs. 4.10%). This could be explained by the usage of an independent (or separate) mobile web version by some sites -that need to canonicalize to their desktop URLs alternates.
+
+Canonical URLs can be specified through different methods, by using the canonical link via the HTTP headers or the HTML head of a page, or by submitting them in XML Sitemaps. When analyzing which is the most popular canonical link implementation method, it was found that only 1.03% of desktop homepages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML head of a page.
+
+{{ figure_markup(
+  image="seo-canonical-implementation-method.png",
+  caption="Canonical Implementation Method",
+  description="Bar chart showing the canonical tag implementation method. We observed that only 1.03% of desktop homepages and 0.88% of mobile ones are relying on the HTTP headers for their implementation, meaning that canonical tags are prominently implemented via the HTML head of a page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=542127514&format=interactive",
+  sheets_gid="337739550",
+  sql_file="pages_wpt_bodies_by_device.sqll"
+  )
+}}
```

Comment:
Also some of the charts have no labels:

![image](https://user-images.githubusercontent.com/10931297/100544042-128b7f00-324b-11eb-8031-9d3bc4e7efcc.png)

Some have both labels:

![image](https://user-images.githubusercontent.com/10931297/100544062-20d99b00-324b-11eb-8393-4407fbfb54c3.png)

Some have both labels with one in outline style:

![image](https://user-images.githubusercontent.com/10931297/100544073-2fc04d80-324b-11eb-80ee-237735aa5266.png)

We should be consistent here. I'd advise both labels if they fit, and only one (usually the larger - so often mobile) if that looks too messy, and not to use outline at all.

Unfortunately that will mean re-exporting the images after you change the chart in sheets 😢 @antoineeripret do you want to take this one if @aleyda agrees?

Reply:
Hi @bazzadp! Thanks so much for catching this. I agree about leaving the two labels if possible and if not, just one. You can go ahead @antoineeripret  :) Thanks again!

Reply:
Note @antoineeripret I changed the image names to remove `fig` + number from the start (makes reordering easier in case we add or remove figures!), add `seo` to the start and also change `_` to `-` as that's what most other chapters use.

So I moved `fig10_meta_description_word_length.png` to `seo-meta-description-word-length.png` for example.

Just in case you get confused when uploading new versions of the images!

Reply:
I got edit access to the Sheet so this is fixed.